doc/advanced-topics.md

```diff
@@ -70,10 +70,10 @@ curl http://${KEYSTONE_HOST}/v3/OS-TRUST/trusts \
 Apart from the generation of the trust, the use of secured Context Brokers should be transparent to the user of the IoT
```

Comment:
I understand this PR provides "the definitive fix" mentioned in https://github.com/telefonicaid/iotagent-node-lib/issues/1012#issuecomment-814014235

Thus, an entry in CHANGES_NEXT_RELEASE relating issue #1012 should be included.